manager/ui/war/plugins/api-manager/ts/api/api-def.ts

```diff
@@ -36,38 +36,66 @@ module Apiman {
                 EntityStatusSvc.getEntity().modifiedBy = CurrentUser.getCurrentUser();
             };
 
-            $scope.saveApi = function() {
+            let isValidDefinition = function (data, definitionType) {

```

Comment:
Think we can use `const` or `$scope.` here - whichever you prefer.

And I think we should prefer `() => {}` syntax nowadays.

Reply:
I change this and will give this another test (I will be careful about UI changes :D).
In general the `() => {}` is not a replacement for all functions. There are some circumstances where you should avoid them: https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/

Reply:
Feel free to leave it if you don't think it's necessary.